app/routes/content.js

```diff
@@ -104,8 +104,17 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated')).toISOString()],
-                ['article:modified_time', new Date(preprint.get('dateModified')).toISOString()]
+                ['article:published_time', d.toISOString()],
+                ['article:modified_time', new Date(preprint.get('dateModified')).toISOString()],
+                ['citation_title', node.get('title')],
+                ['citation_description', node.get('description')],
+                ['citation_public_url', window.location.href],
+                ['citation_publication_date', `${d.getFullYear()}/${d.getMonth() + 1}/${d.getDate()}`],
+                ['citation_doi', preprint.get('doi')],
+                ['dc.title', node.get('title')],
+                ['dc.abstract', node.get('description')],
+                ['dc.identifier', window.location.href],
```

Comment:
Same here on getting this from the API if it's available. And should there be two `dc.identifier` fields?